src/components/lobby/LobbyAwaitGame.tsx

```diff
@@ -0,0 +1,222 @@
+// tslint:disable:no-console
+import * as React from 'react';
+import styled from 'styled-components';
+import BGImage from '../../dist/images/chess-bg.jpg';
+import {CreatedGames} from '../../store/MockData';
+
+const ServerData = CreatedGames;
+const mockGame = {creator : "Anonymous", creatorColor: "",  opponent : "Anonymous", gameId : 2, gameType : "Classic", gameTime : 5, gameAddTime : 15, gameStarted : false, player1Ready: false, player2Ready: false,}; 
```

Comment:
Would be nice if the mock-data was moved to an external file - and an interface-method was created in a store that fetches the data.

That way this component will be more complete, and further changes to the data-flow does not need to alter LobbyAwaitGame.

My suggestion is that we use LobbyStore / and create a method for fetching the data through an observable, as is done with GameStore. @Filipo0on

Reply:
Done